src/coordinated_workers/nginx.py

```diff
@@ -228,6 +228,10 @@ class NginxLocationConfig:
     """Whether to connect to the upstream over TLS (e.g., https:// or grpcs://)
     If None, it will inherit the TLS setting from the server block that the location is part of.
     """
+    rewrite: Optional[List[str]] = None
```

Comment:
just an API thought, but wouldn't it feel more natural to ask for a simple `str` and handle the `.split()` in our code instead of asking for a pre-split string?